internal/resources/fabric/stream_subscription/models.go

```diff
@@ -0,0 +1,385 @@
+package streamsubscription
+
+import (
+	"context"
+
+	"github.com/equinix/terraform-provider-equinix/internal/converters"
+	"github.com/equinix/terraform-provider-equinix/internal/fabric"
+	int_fw "github.com/equinix/terraform-provider-equinix/internal/framework"
+	fwtypes "github.com/equinix/terraform-provider-equinix/internal/framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+
+	"github.com/equinix/equinix-sdk-go/services/fabricv4"
+
+	"github.com/hashicorp/terraform-plugin-framework-timeouts/resource/timeouts"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+type DataSourceByIDsModel struct {
+	ID             types.String `tfsdk:"id"`
```

Comment:
Nit / FYI: there is no need to export model types; if you start them with a lower-case letter they will only be accessible within the resource package, which reduces the risk of us incorrectly or unexpectedly tying different resource packages together.